app/[locale]/events.tsx

```diff
@@ -0,0 +1,205 @@
+            },
+
+            {
+              imageSrc: 'slideshow/image01.jpg',
+              title: 'New Library Inauguration!',
+              description:
+                'The new library at Nit Kurukshetra was inaugurated with modern facilities and a large collection of books to enhance the learning experience for students.',
+            },
+            {
+              imageSrc: 'slideshow/image01.jpg',
+              title: 'Alumni Meet 2024!',
+              description:
+                'Nit Kurukshetra hosted its annual alumni meet, welcoming graduates from various batches to reconnect and share their experiences with current students.',
+            },
+            {
+              imageSrc: 'slideshow/image01.jpg',
+              title: 'Tech Fest 2024 Announced!',
+              description:
+                'The much-awaited Tech Fest 2024 is coming soon, promising a lineup of exciting workshops, hackathons, and competitions for tech enthusiasts.',
+            },
+            {
+              imageSrc: 'slideshow/image01.jpg',
+              title: 'Campus Placement Success!',
+              description:
+                'The placement season at Nit Kurukshetra has been a huge success, with many students securing offers from top companies in various industries.',
+            },
+            {
+              imageSrc: 'slideshow/image01.jpg',
+              title: 'Cultural Fest Brings Vibrancy!',
+              description:
+                'The Cultural Fest at Nit Kurukshetra brought vibrant performances, music, and dance, showcasing the diverse talents of the students.',
+            },
+          ].map(
+            ({ imageSrc, title, description }, index, arr) =>
+              index % 2 === 0 && (
+                <section
+                  key={index}
+                  className={cn(
+                    'gap-6 lg:grid',
+                    index % 4 === 0
+                      ? 'grid-cols-[63%,37%]'
+                      : 'grid-cols-[37%,63%]'
+                  )}
+                >
+                  <Card className="relative border-0 bg-shade-light">
+                    <Image
+                      alt={'meow'}
+                      className="h-64 w-full rounded-t-md bg-neutral-900 object-cover"
+                      height={0}
+                      src={imageSrc}
+                      width={0}
+                    />
+                    <section className="absolute bottom-[35%] right-0 mr-4 mt-4 flex h-20 w-16 flex-col items-center justify-center bg-primary-500 bg-opacity-90 text-shade-light">
+                      <span className="font-serif text-2xl">02</span>
+                      <span className="font-serif text-xl">SEP</span>
```

Comment:
Just add `eventDate` or similar to the object
```suggestion
                      <span className="font-serif text-2xl">{new Intl.DateTimeFormat('en-GB', { day: '2-digit' }).format(eventDate)}</span>
                      <span className="font-serif text-xl">{capitalise(new Intl.DateTimeFormat('en-GB', { month: 'short' }).format(eventDate).substring(0, 3))}</span>
```